settings.gradle.kts

```diff
@@ -99,14 +99,19 @@ gradleEnterprise {
     }
 }
 
+// The HAPI API version to use for Protobuf sources. This can be a tag or branch
+// name from the hedera-protobufs GIT repo.
+val hapiProtoVersion = "0.40.0-blocks-state-SNAPSHOT"
+val hapiProtoBranchOrTag = "add-pbj-types-for-state" // hapiProtoVersion
```

Comment:
Not sure what we want here, but if we set the version directly to the branch/tag name, we have the following issue:

[hapiProtoBranchOrTag](https://scans.gradle.com/s/5ow2f4cief7zo/tests/:hedera-node:test-clients:itest/AllIntegrationTests/logValidation()%5B1%5D?focused-exception-line=0-3&top-execution=1)

```
WARN  90   SemanticVersions - Failed to parse resource 'semantic-version.properties' (keys 'hapi.proto.version' and 'hedera.services.version') and resource 'bootstrap.properties' (key 'hedera.config.version'). Version info will be unavailable!
java.lang.IllegalArgumentException: Argument value='add-pbj-types-for-state' is not a valid semver
```